versions/3.0.md

```diff
@@ -176,7 +176,7 @@ In the following description, if a field is not explicitly **Required** or descr
 #### <a name="oasObject"></a>OpenAPI Object
 
 This is the root document object for the API specification.
-It combines what previously was the Resource Listing and API Declaration (version 1.2 and earlier) together into one document.
+The OpenAPI object combines the Resource Listing and API Declaration from version 1.2 and earlier into one document.
```

Comment:
@krishahn this sentence is actually obsolete now (it was relevant in 2.0). Mind removing it altogether?